networkmanager/networkmanager.go

```diff
@@ -208,6 +217,28 @@ func New(cfg *config.Config, trafficStorage TrafficStorage) (manager *NetworkMan
 		if err != nil {
 			return manager, err
 		}
+
+		manager.pollTimer = time.NewTicker(UpdateIptablesCachePeriod)
+		ctx, cancelFunc := context.WithCancel(context.Background())
+		manager.cancelFunction = cancelFunc
+
+		go func(ctx context.Context) {
+			for {
+				select {
+				case <-ctx.Done():
+					return
+
+				case <-manager.pollTimer.C:
+					if err := manager.trafficMonitoring.processTrafficMonitor(); err != nil {
+						log.Errorf("Failed to process traffic monitor: %v", err)
+					}
+
+					if err := manager.trafficMonitoring.updateIptablesFilterCache(); err != nil {
```

Comment:
We should first update cache then process traffic, shouldn't we?

Reply:
Done